teamtalk/instance.py

```diff
@@ -138,6 +139,126 @@ def change_status(self, status_mode: UserStatusMode, status_message: str):
         """
         self.super.doChangeStatus(status_mode, sdk.ttstr(status_message))
 
+    def get_sound_devices(self) -> List[SoundDevice]:
+        """Gets the list of available TeamTalk sound devices.
+
+        Calls TT_GetSoundDevices from the SDK.
+
+        Returns:
+            A list of SoundDevice objects representing the available devices.
+            Returns an empty list if the SDK call fails.
+        """
+        count = ctypes.c_int()
+        success = sdk._GetSoundDevices(None, ctypes.byref(count))
```

Comment:
This will probably not work as expected. Also for simplying it and keep it consistant with other functions, just use self.super.getSoundDevices in this function and then convert the result to the TeamTalkSoundDevice classes